### DISCUSSION TITLE
Adopter should have access to disable Dialcode context feature

### DISCUSSION THREAD

**Discussion by amitpriyadarshi** [Sunbird-Knowlg/Community#82] [Feature Requests]
@vinukumar-vs @AmiableAnil 
As part of dialcode context feature, there is a config property in content publish job named `enableDIALContextUpdate`. By default its value is hardcoded as `Yes`, which mean any adopter can not disable this feature.

Please variablize it so that we can update its value accordingly.

**Comment by Jayaprakash8887** [comment]
@amitpriyadarshi Adopters always have the option to maintain their own copies of configurations and use them at the time of deployment instead of sunbird configurations.
